src/tests/ftest/rebuild/rebuild_container_rf.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2019-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from container_rf_test_base import RbldContRedundancyFactor
+
+
+class RbldContRedundancyFactorrr(RbldContRedundancyFactor):
```

Comment:
Is this a typo? RbldContRedundancyFactorrr

Reply:
Thanks for the comment, please see next commit for the update.